src/renderer/components/recording-page/recording-visualization.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { mocked } from 'ts-jest/utils';
+import WaveSurfer from 'wavesurfer.js';
+
+import { RecordingVisualization } from './recording-visualization';
+jest.mock('wavesurfer.js');
+
+// Mock the remote module to prevent import failure.
+jest.mock('@electron/remote', () => ({ dialog: jest.fn() }));
```

Comment:
Did you check if this line is applicable to your test?

Reply:
yes the test suite fails to run without that line